pytorch_lightning/profiler/base.py

```diff
@@ -180,15 +195,6 @@ def teardown(self, stage: Optional[str] = None) -> None:
     def __del__(self) -> None:
         self.teardown(stage=self._stage)
 
-    def start(self, action_name: str) -> None:
-        raise NotImplementedError
-
-    def stop(self, action_name: str) -> None:
-        raise NotImplementedError
-
-    def summary(self) -> str:
```

Comment:
summary can return an empty string instead of raising a NotImplementedError. this way the passthrough profiler doesn't need to override this, and users who don't implement this won't see exceptions